docs/files/Mixpanel.php.txt

```diff
@@ -192,11 +192,12 @@ class Mixpanel extends Base_MixpanelBase {
 
 
     /**
-     * Identify the user you want to associate to tracked events
+     * Identify the user you want to associate to tracked events. The $anon_id must be UUID v4 format and not already merged to an $identified_id.
```

Comment:
can we expand on this doc to explain the two different use-cases for the function? I just want to make it really clear for users

Reply:
yessir, and i shall learn. how. to. write. p.h.p tests